app/views/volunteer_ops/index.html.erb

```diff
@@ -16,11 +16,19 @@
       <% @volunteer_ops.each do |op| %>
           <tr>
             <td>
-              <strong><%= link_to op.title, op.link %></strong><br/>
-              <div id="organisations">
-                <small><%= link_to op.organisation_name, op.organisation_link %></small>
-                <br/>
-              </div>
+              <% unless op.source == "local" %>
```

Comment:
this involves a bit of replication - I wonder if we can pull the logic into a helper?  or check out http://stackoverflow.com/a/11874485/316729  - I can imagine doing something here like a helper that would be

```rb
def link_to_vol_op(op, html_options = {})
  html_options.merge(target: "_blank") unless op.source == 'local' 
  link_to(op.organisation_name, op.organisation_link, html_options)
end
```

which might even DRY out some other places too?